src/main/java/stringCalculator/StringCalculator.java

```diff
@@ -0,0 +1,30 @@
+package stringCalculator;
+
+public class StringCalculator {
+    public int plusCalculate(String[] numbers) {
+        int result = 0;
+        for (String number : numbers) {
+            checkNegativeNumberOrNotNumber(number);
+            result += Integer.parseInt(number);
+        }
+        return result;
+    }
+
+    public void checkNegativeNumberOrNotNumber(String inCorrectNumber) {
+        if(Integer.parseInt(inCorrectNumber)<0){
+            throw new IllegalArgumentException("0 이상의 수를 입력하세요.");
+        }
+    }
+
+    public int calculate(String inputString) {
+        if(isEmptyOrNull(inputString)){
```

Comment:
```suggestion
        if (isEmptyOrNull(inputString)) {
```
0도 매직넘버 추출 되면 좋을 것 같습니다 😄 
이번 과정 진행 하시면서 이런 부분도 좀 더 엄격하게 연습해보시는걸 추천 드립니다 🙇